patient_portal/messaging/messages.php

```diff
@@ -110,11 +109,7 @@ function getAuthPortalUsers()
 <!--<script type='text/javascript' src="./appmail.js.php"></script>  -->
 
 </head>
-<!-- style='background:#f1f2f7;' -->
-<body class="body_top">
-<div class='title'>
-        <h2><img style='width:25%;height:auto;' class='logo' src='<?php echo $GLOBALS['images_path']; ?>/logo-full-con.png'/>  <?php echo xlt('Patient Messaging'); ?></h2>
-        </div>
```

Comment:
@ahanipeitra1994 it looks like with your last commit, you removed the solution you pushed earlier, so now the original issue still exists. Make sure to always test before you push!